modules/cms-kit/src/Volo.CmsKit.Admin.Web/Pages/CmsKit/Contents/addWidgetModal.js

```diff
@@ -48,16 +43,8 @@ $(function () {
                     updatedText += "]";
                 }
 
-                if (contentEditorText == '\n\n\n') {
-                    //TODO fails event
-                    var fixedData = "<div>" + updatedText + "</div>";
-                    var innerHtml = $("#ContentEditor")[0].innerHTML;
-                    var replacedInnerHtml = innerHtml.replace('<div><br></div>', fixedData);
-                    $("#ContentEditor")[0].innerHTML = replacedInnerHtml;
-                }
-                else {
-                    $('.ProseMirror div').contents()[0].data = contentEditorText + updatedText;
-                }
+                $('#GeneratedWidgetText').val(updatedText);
+                $("#GeneratedWidgetText").trigger("click");
```

Comment:
We can use 'change' event instead of 'click'. 

In that case, it's tricky and hard to understand what happens. At the first time, it feels like user clicks on it.